config/generic-mks-sgenl-v2.cfg

```diff
@@ -0,0 +1,216 @@
+sense_resistor: 0.110
+stealthchop_threshold: 0
+#####################################################################
+#Extruder
+#####################################################################
+[extruder]
+step_pin: P2.13
+dir_pin: !P0.11
+enable_pin: !P2.12
+## Update value below when you perform extruder calibration
+## If you ask for 100mm of filament, but in reality it is 98mm:
+## rotation_distance = <previous_rotation_distance> * <actual_extrude_distance> / 100
+rotation_distance: 23.46041055718
+max_extrude_only_distance: 110.0
+microsteps: 16
+full_steps_per_rotation: 200    #200 for 1.8 degree, 400 for 0.9 degree
+nozzle_diameter: 0.400
+filament_diameter: 1.75
+heater_pin: P2.7
+## Validate the following thermistor type to make sure it is correct
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: P0.23
+min_temp: 10
+max_temp: 270
+max_power: 1.0
+control = pid
+pid_kp = 20.741
+pid_ki = 0.981
+pid_kd = 109.667
+
+
+[tmc2209 extruder]
+uart_pin: P1.15
+run_current: 0.3
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+
+#####################################################################
+# Bed Heater
+#####################################################################
+
+[heater_bed]
+# SSR Pin - BED_OUT
+heater_pin: P2.5
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: P0.24
+control: watermark
+min_temp: 0
+max_temp: 110
+
+
+#####################################################################
+# Probe
+#####################################################################
+
+#[probe]
+#pin: ^!P1.25
+#x_offset: -10
+#y_offset: 0
+#z_offset: 1
+#speed: 15.0
+#lift_speed: 30.0
+#samples: 3
+#samples_result: average
+#sample_retract_dist: 2.0
+#samples_tolerance: 0.050
+#samples_tolerance_retries: 8
+#####################################################################
+# Fan Control
+#####################################################################
+
+[fan]
+pin: P2.4
+off_below: 0.10
+
+[heater_fan hotend_fan]
+pin: P1.4
+max_power: 1.0
+fan_speed: 1.0
+heater: extruder
+heater_temp: 60.0
+
+[safe_z_home]
+home_xy_position:10,10
+speed:350
+z_hop:3
+
+
+#--------------------------------------------------------------------
+########################################
+# EXP1 / EXP2 (display) pins
+########################################
+
+[board_pins]
+aliases:
+    # EXP1 header
+    EXP1_1=P1.31, EXP1_3=P0.18, EXP1_5=P0.15, EXP1_7=P1.0,  EXP1_9=<GND>,
+    EXP1_2=P1.30, EXP1_4=P0.16, EXP1_6=P0.17, EXP1_8=P1.22, EXP1_10=<5V>,
+
+    # EXP2 header
+    EXP2_1=P0.8, EXP2_3=P3.25, EXP2_5=P3.26, EXP2_7=P0.27, EXP2_9=<GND>,
+    EXP2_2=P0.7, EXP2_4=P0.28, EXP2_6=P0.9,  EXP2_8=<RST>, EXP2_10=<NC>
+
+#--------------------------------------------------------------------
+
+[pause_resume]
+
+##   Thermistor Types
+##   "EPCOS 100K B57560G104F"
+##   "ATC Semitec 104GT-2"
+##   "NTC Generic 3950"
+##   "Honeywell 100K 135-104LAG-J01"
+##   "NTC 100K MGB18-104F39050L32" (Keenovo Heater Pad)
+##   "AD595"
+##   "PT100 INA826"
```

Comment:
Not in a generic board config

Reply:
my mistake,
it would be in the "[include fluidd.cfg]" in my scenario.